ci/jenkins/test.sh

```diff
@@ -534,10 +556,13 @@ function deliver_antrea {
     export GOROOT=${GOLANG_RELEASE_DIR}/go
     export GOCACHE="${WORKSPACE}/../gocache"
     export PATH=${GOROOT}/bin:$PATH
-
     git show --numstat
     make clean
-    ${CLEAN_STALE_IMAGES}
+    if [[ $TESTBED_TYPE != "kind" ]]; then
+       if [[ $TESTBED_TYPE != "kind-flexible-ipam" ]]; then
+          ${CLEAN_STALE_IMAGES}
```

Comment:
Is this related to the rate limit issue? Why only clean the images for the IPAM kind testbed?

Reply:
yes, we no need run this cmd "docker system prune --force --all --filter until=4h" to clean available images for kind and kind-flexible-ipam.